EnglishFilter/sections/specific.txt

```diff
@@ -435,7 +435,8 @@ upload-4ever.com##a[onclick^="window.open("] ~ a[style*="text-align: left;"][sty
 ||banner-api.grouphfm.com/banner/
 ||zoophiliahentai.com/uploads/banners/
 olx.ro,olx.kz,olx.pl,olx.pt,olx.ua,olx.uz,olx.in,olx.com##div[style="min-height: 120px; display: block;"]
-http$third-party,~image,domain=imageking.xyz|imgkoi.xyz|imgair.net|imgko.xyz|imglin.xyz|imgtigr.xyz
+http$third-party,~image,domain=imageking.xyz|imgkoi.xyz|imgair.net|imgko.xyz|imglin.xyz|imgtigr.xyz|imgkr.xyz
+||imgkr.xyz/shrinker/js/h1j.js
 ||imgtigr.xyz/shrinker/js/h1j.js
 ||imglin.xyz/shrinker/js/h1j.js
 ||imgair.net/shrinker/js/h1j.js
```

Comment:
You can create a common rule like `/shrinker/js/h1j.js$domain=` with necessary domains.

Reply:
Don't need.

Reply:
Why not? This is more useful with a single rule than creating a bunch of similar ones.

Reply:
-_- Okay fine. Wait up.